python/marvin/contrib/vacs/mangaVM.py

```diff
@@ -0,0 +1,92 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Licensed under a 3-clause BSD license.
+#
+# @Author: Brian Cherinka
+# @Date:   2018-10-11 17:51:43
+# @Last modified by:   Brian Cherinka
+# @Last Modified time: 2018-11-29 17:23:15
+
+from __future__ import print_function, division, absolute_import
+
+import numpy as np
+import astropy
+import astropy.units as u
+import marvin.tools
+from marvin.tools.quantities.spectrum import Spectrum
+from marvin.utils.general.general import get_drpall_table
+from marvin.utils.plot.scatter import plot as scatplot
+
+import matplotlib.image as mpimg
+import matplotlib.pyplot as plt
+
+from .base import VACMixIn
+
+
+class VMORPHOVAC(VACMixIn):
+    """Provides access to the MaNGA-VISUAL-MORPHOLOGY VAC.
+
+    VAC name: manga_visual_morpho
+
+    URL: link
+
+    Description: Returns visual morphology data
+
+    Authors: J. Antonio Vazquez-Mata and Hector Hernandez-Toledo
+
+    """
+
+    # Required parameters
+    name = 'mangaVM'
+    description = 'Returns visual morphology data'
+    version = {'MPL-7': '1.0.1',
+               'DR15': '1.0.1'}
+
+    # optional Marvin Tools to attach your vac to
+    include = (marvin.tools.cube.Cube,
+               marvin.tools.maps.Maps)
+    #           marvin.tools.modelcube.ModelCube)
+
+    # Required method
+    def get_data(self, parent_object):
+
+        # get any parameters you need from the parent object
+        plateifu = parent_object.plateifu
+        release = parent_object.release
+
+        # define the variables to build a unique path to your VAC file
+        path_params = {'vmver': self.version[release],
+                       'plateifu': plateifu, 'survey': '*'}
+
```

Comment:
Since you are trying to get the path for both sdss and desi images, the `path_params` here should probably be explicit with `survey`. See my comment below.

Reply:
if I understand correctly, this should be something like this?
'survey' : ['sdss', 'desi']

Reply:
Sorry for the long delay.  You will need to create two dictionaries, one for `survey: 'sdss'` and one for `survey: 'desi'`.  These `path_params` dictionaries are used to create the explicit path to your files.

Reply:
Sorry, I was away on holidays for a few days. I submitted a new version in my fork where, I dont use "img" to show the mosaic. Instead I just want the mosaics to be downloaded automatically when the user requires a galaxy information from this VAC.